### PR TITLE
[Hexagon] Add tests for 2 operators that utilize Qualcomm HVX intrinsics.

### PR DESCRIPTION
**Pull request by nverke**
**Background**
This PR adds 2 tests for unused HVX implementations of mmul and conv2d. These were created to experiment with the performance of HVX intrinsics in tvm and each test outputs the runtime of each on various different input shapes. These will also be a good reference point for others looking to start using HVX intrinsics in their schedules. 

**Testing**
There are no code changes here outside of the tests themselves.

cc @mehrdadh @csullivan

### CODE REVIEW

File: tests/python/contrib/test_hexagon/test_mmul_hvx_intrinsics.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        out = c_hexagon.numpy()
+        out_a = out.reshape(batches * n * k)
+        out_req, _, _ = quantize_array(out_a, batches * n * k)
+        out_req = np.array(out_req).reshape(batches, n, k)
+
+        out_ref_a = out_ref.reshape(batches * n * k)
+        out_ref_q, _, _ = quantize_array(out_ref_a, batches * n * k)
+        out_ref_q = np.array(out_ref_q).reshape(batches, n, k)
+
+        tvm.testing.assert_allclose(out_req, out_ref_q, atol=2.0, rtol=0.0)
```
Comment (mehrdadh) [inline tests/python/contrib/test_hexagon/test_mmul_hvx_intrinsics.py:205]: `tvm.testing.main()` is missing

**Review by mehrdadh — CHANGES_REQUESTED** [review]
I suggest we move these files into `tests/python/contrib/test_hexagon/hvx`. wdyt?

File: tests/python/contrib/test_hexagon/test_conv_hvx_intrinsics.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        (create_inputs((2, 34, 34, 29), (5, 5, 29, 2), "local")),
+        (create_inputs((1, 512, 512, 1), (9, 9, 1, 1), "local")),
+    )
+
+    @tvm.testing.requires_hexagon
+    def test_vrmpy_conv(
+        self, hexagon_session, a, w, a_offset, w_offset, expected_output, mem_scope
+    ):
+
+        # TODO even sized kernels and stride are currently not working.
```
Comment (mehrdadh) [inline tests/python/contrib/test_hexagon/test_conv_hvx_intrinsics.py:75]: please change this to: `TODO(your github handler): ...` format

File: tests/python/contrib/test_hexagon/test_mmul_hvx_intrinsics.py (hunk 1 above)
Comment (nverke) [reply tests/python/contrib/test_hexagon/test_mmul_hvx_intrinsics.py:205]: Added it in!

File: tests/python/contrib/test_hexagon/test_conv_hvx_intrinsics.py (hunk 2 above)
Comment (nverke) [reply tests/python/contrib/test_hexagon/test_conv_hvx_intrinsics.py:75]: Done!

File: tests/python/contrib/test_hexagon/conv_uint8.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
```
Comment (mehrdadh) [inline tests/python/contrib/test_hexagon/conv_uint8.py:3]: any reason we should keep these files(conv_uint8.py and conv_uint8_hvx...) here and not in the hvx directory?

**Pull request closed without merging**